include/crab/analysis/inter/top_down_inter_analyzer.hpp

```diff
@@ -419,6 +420,13 @@ class global_context {
   using func_fixpoint_map_t =
       std::unordered_map<callgraph_node_t, func_fixpoint_map_entry_t>;
 
+  using variable_vector_t = std::vector<variable_t>;
+  using region_equiv_class_t = std::vector<variable_vector_t>;
+  using callsite_t = typename cfg_t::basic_block_t::callsite_t;
+  using callsite_info_t = domains::callsite_info<variable_t>;
+  using callsite_dsa_info_map_t =
```

Comment:
Just call it `callsite_info_map_t`. I want to avoid the mention to "dsa". The idea is to keep information about the callsite. For now, we only care about one dsa-specific intrinsic but this might contain non-dsa stuff in the future